### DISCUSSION TITLE
Warning text on Select?

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#958] [Q&A] [answered]
Hello, 
I'd like to know if I can use a warning text under the Select component. 
<img width="1060" alt="Screen Shot 2020-11-18 at 5 10 32 PM" src="https://user-images.githubusercontent.com/67388401/99607676-1c96bd80-29c1-11eb-97b2-9d379d92d772.png"> as you see here, Case 1 shows an error message that relates to an error but for me Case 2 is more like a warning instead of an error. 

If Case 2 is an error. I will use the design above. If it's a warning, should I use a warning text instead?

**Comment by serifluous** [comment, marked as answer]
Hey @sang-yeo ! I'm going to rephrase your question (just so you can make sure I fully understand it): Are you asking what you should do in this case where the form field isn't _exactly_ in error, but you need a way to tell the user that they've hit the maximum search query value (and they can't enter any more values)?

^ If that's the case, I agree that error text isn't quite right. This isn't a very common thing to do, but you can compose the warning text you need by putting in any Icon and Text component in a form field's Help Text.

**Reply by sang-yeo** [reply]
Yes and thank you for your quick answering!

**Reply by sang-yeo** [reply]
<img width="906" alt="Screen Shot 2020-11-18 at 6 22 38 PM" src="https://user-images.githubusercontent.com/67388401/99612769-13aae980-29cb-11eb-9bd0-fdf90f2e9902.png">

@serifluous something like this?

**Reply by serifluous** [reply]
Yep, if that's not obvious enough, you can also change the text color, too (you'll have to use $color-text-warning-dark for it to be accessible).

The interaction might be tricky to get right here, too, since it's not very common in console (as far as I know), so it'll help to take this through usability testing or more design crits.
